module-sys/SystemWatchdog/include/SystemWatchdog/SystemWatchdog.hpp

```diff
@@ -57,10 +57,6 @@ namespace sys
         bool timeout_occurred{false};
         bool enableRunLoop{false};
         cpp_freertos::BinarySemaphore taskEndedSem{false};
-
-        static_assert(sizeof(lastRefreshTimestamp) == 4 && alignof(decltype(lastRefreshTimestamp)) == 4,
```

Comment:
It was removed mainly for two reasons:
1. Simulator port uses different size for `TickType_t`, 8 bytes in this case, hence this code is not portable.
2. This check is redundant. By the FreeRTOS design, `TickType_t` is always of atomic type. It's because its size is always the size of architecture(4 bytes for 32bit architecture, 2 bytes for 16bit, and so on).